### PR TITLE
prev&next btns + remember btn styles added

### PR DESCRIPTION
**Pull request by olegmikadze**

### CODE REVIEW

**Comment by YaroslavNikiforov** [comment]
Please, align them vertically and horizontally, remove border, because it does not work anyway (or use !important), and them can be not dublicated, because of the same properties (&-tab-prev, &-tab-next {...)

File: front-end/src/components/Login/index.js [hunk 1]
```diff
@@ -51,6 +51,10 @@ class Login extends Component {
             <label htmlFor="password">Пароль</label>
             <input type="password" name="password" id="password" placeholder="введите пароль (1234)"
               value={this.state.password} onChange={this.handleChange} required/>
+            <div className="remember">
```
Comment (YaroslavNikiforov) [inline front-end/src/components/Login/index.js]: class name must begin with block name

Comment (olegmikadze) [reply front-end/src/components/Login/index.js]: I've tried to do like BEM but the checkbox, didn't work properly, that way((

Comment (YaroslavNikiforov) [reply front-end/src/components/Login/index.js]: +

**Pull request merged**